internal/provider/provider.go

```diff
@@ -129,6 +128,7 @@ func (p *BloxOneProvider) Resources(_ context.Context) []func() resource.Resourc
 
 		dfp.NewDfpResource,
 
+		fw.NewSecurityPoliciesResource,
 		fw.NewAccessCodesResource,
 		fw.NewNamedListsResource,
 		fw.NewNetworkListsResource,
```

Comment:
nit: not for this pr, but all of these resource names here should be "singular".

Reply:
Will fix in a mis PR with other changes